### PR TITLE
Add README following the template

### PR DESCRIPTION
**Pull request by arbroween**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Ô Chap'
+
+Ô Chap' is an e-commerce website for selling all kinds of hats!
+
+## Installation
+
+### Requirements
+
+- Git
+- JDK 1.8 or later
```
Comment (roosemberth) [inline README.md:10]: The project's POM requests [java 11](https://github.com/AMT-2021/AMT/blob/master/pom.xml#L17).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Instructions
+
+```bash
+# Go to the directory where you want to clone the project.
+cd path/where/to/put/the/project
+
+# Clone the project.
+git clone https://github.com/AMT-2021/AMT
+
```
Comment (roosemberth) [inline README.md:21]: Not really useful, I think it should be removed.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# Run the project locally with Maven.
```
Comment (roosemberth) [inline README.md:22]:
```suggestion
# Run the project locally with Maven using spring's built-in development server.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+You should be able to access the website on [http://localhost:8080/](http://localhost:8080/).
+
+## Usage
+
+Once the project is running locally, you can access it with a web browser on [http://localhost:8080/](http://localhost:8080/).
+
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
```
Comment (roosemberth) [inline README.md:33]: What is a major change? An example of something more precise would be something along the lines of: _pull requests for bugfixes are very welcome. For new features or other changes, please open an issue first._

**Review by roosemberth — CHANGES_REQUESTED** [review]
Thanks for your contribution, I've left some remarks.

**Comment by arbroween** [comment]
I fixed the pull request based on the feedback

File: README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+### Instructions
+
+Clone the current repository.
+
+Run the project locally with Maven using Spring's built-in development server:
+```bash
+mvn spring-boot:run
+```
+
+You should be able to access the website on [http://localhost:8080/](http://localhost:8080/).
```
Comment (roosemberth) [inline README.md:22]: This endpoint does not exist atm, but I think it's fine since we expect to include it soon.

**Review by roosemberth — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**